src/tests/ftest/util/fio_test_base.py

```diff
@@ -79,31 +86,30 @@ def _create_pool(self):
         # Create a pool
         self.pool.create()
 
-    def _create_cont(self):
-        """Create a TestContainer object to be used to create container."""
-        # TO-DO: Enable container using TestContainer object,
-        # once DAOS-3355 is resolved.
-        # Get Container params
-        # self.container = TestContainer(self.pool)
-        # self.container.get_params(self)
-
-        # create container
-        # self.container.create()
-        env = Dfuse(self.hostlist_clients, self.tmp).get_default_env()
-        # command to create container of posix type
-        cmd = env + "daos cont create --pool={} --svc={} --type=POSIX".format(
-            self.pool.uuid, ":".join(
-                [str(item) for item in self.pool.svc_ranks]))
-        try:
-            container = subprocess.Popen(cmd, stdout=subprocess.PIPE,
-                                         shell=True)
-            (output, err) = container.communicate()
-            self.log.info("Container created with UUID %s", output.split()[3])
+    def _create_cont(self, doas_cmd):
+        """Create a container.
+
+        Args:
+            daos_cmd (DaosCommand): doas command to issue the container
+                create
+
+        Returns:
+            str: UUID of the created container
 
-        except subprocess.CalledProcessError as err:
-            self.fail("Container create failed:{}".format(err))
+        """
+        cont_type = self.params.get("type", "/run/container/*")
+        result = self.daos_cmd.container_create(
+            pool=self.pool.uuid, svc=self.pool.svc_ranks,
+            cont_type=cont_type)
 
-        return output.split()[3]
+        # Extract the container UUID from the daos container create output
+        cont_uuid = re.findall(
+            "created\s+container\s+([0-9a-f-]+)", result.stdout)
```

Comment:
(lint) '\s'. String constant might be missing an r prefix.
(lint) '\s'. String constant might be missing an r prefix.